backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -144,7 +144,8 @@ def assign_next(self) -> None:
         reviewers = self.assignees.filter(role=ListRole.reviewer).order_by("order")
 
         # All reviewers have reviewed the draft destruction list
-        if self.assignee == reviewers.last().user:
+        last_reviewer = reviewers.last()
+        if last_reviewer and self.assignee == reviewers.last().user:
             self.get_author().assign()
             status = (
                 ListStatus.ready_to_delete
```

Comment:
I think that this is not correct anymore :see_no_evil: This is only the case for the short procedure :thinking: Otherwise it should be "internally_reviewed" I think

Reply:
Isn't that part of another story/pr?

Reply:
It is in main :thinking: https://github.com/maykinmedia/open-archiefbeheer/blob/8c2ce8a8b2da1660253ec2f0d16b5ca8fc82cfb1/backend/src/openarchiefbeheer/destruction/models.py#L152

Reply:
There is an if statement on the next line: https://github.com/maykinmedia/open-archiefbeheer/blob/6804600874729e1f9c4767889dc64a32ec28187b/backend/src/openarchiefbeheer/destruction/models.py#L152

Reply:
Line 152 contains an if statment branching based on the short procedure

Reply:
I guess GH cache?

Reply:
Huh, now I see it :shrug: strange!